addons/interaction/stringtable.xml

```diff
@@ -720,6 +720,9 @@
             <Turkish>Menzilde Değil</Turkish>
             <Hungarian>Hatótávolságon kívül</Hungarian>
         </Key>
+        <Key ID="STR_ACE_Interaction_OpenBackpack">
+            <English>Open %1</English>
+        </Key>
```

Comment:
There is a vanilla stringtable entry for "open", `STR_single_open`. That could be used instead, but I went with this in case some languages require a specific format

Reply:
STR_ACTION_OPEN_BAG is `"Open %1"`

Reply:
Missed that in the stringtable viewer